AssetInformationListener.Tests/AwsIntegrationTests.cs

```diff
@@ -0,0 +1,80 @@
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.DataModel;
+using Amazon.DynamoDBv2.Model;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace AssetInformationListener.Tests
+{
+    // TODO - Alter if DynamoDb is not required
+
+    public class AwsIntegrationTests
+    {
+        private readonly AwsMockApplicationFactory _factory;
+        private readonly IHost _host;
+
+        public IDynamoDBContext DynamoDbContext => _factory?.DynamoDbContext;
+        private readonly List<TableDef> _tables = new List<TableDef>
+        {
+            new TableDef { Name = "SomeTable", KeyName = "id", KeyType = ScalarAttributeType.S }
```

Comment:
The tablename should be AssetInformation

Reply:
It will be when the next ticket is done. This is just setting up the application shell.